app/views/smart_answers/_previous_answers.html.erb

```diff
@@ -2,7 +2,12 @@
   <table class="govuk-table previous-answers-table">
     <caption class="govuk-table__caption">
       <h2 class="govuk-heading-m">Previous answers</h2>
-      <p class="govuk-body"><%= link_to "Start again", restart_flow_path(@presenter), :class => "govuk-link" %></p>
+      <p class="govuk-body"><%= link_to "Start again", restart_flow_path(@presenter), :class => "govuk-link", :data => {
+        module: "track-click",
+        "track-action": "Start again",
+        "track-category": "StartAgain",
```

Comment:
Do we need to check if a PA if these labels are suitable?

Reply:
This is what it said on the spreadsheet (in the trello)